### ISSUE TITLE
Test best model for current dataset & smaller dataset & compare

### ISSUE THREAD

**Issue by malteserteresa** [opt-out-tool/opt-out#2]

**Comment by MikeJohnPage** [comment]
For selecting the correct model, the Wiki 'modelling' page is slowly curating potential models, features, and considerations. Choosing the best model is determined by a multitude of factors including the number of training samples, the dimensions of the feature space, and the nature of the data.

I did a quick scan and for neural nets it seems you need approximately a minimum of [50 times](https://www.sciencedirect.com/science/article/pii/S1755534518300058) the number of weights. Nonetheless fine tuning of the parameters may allow a neural net to be maximised on [smaller samples](https://beamandrew.github.io/deeplearning/2017/06/04/deep_learning_works.html).

As the project scales and the sample size grows (hopefully with a larger sized training data set on its way - watch this space), the modelling considerations will invariably change. I argue at this stage, while evaluating a proof of concept, the neural net be kept and refined. Later, a systematic evaluation of different models be conducted once the project has its feet and the training data is suffice.

**Comment by MikeJohnPage** [comment]
Also, if you do want to try different models at this stage, SVM can obtain decent results on small samples (although requires fine tuning of the parameters). The best strategy, however, will be to run and evaluate all of the models listed in the 'modelling' Wiki for their F-scores (or some other common error metric).
